packages/taikoon-ui/src/routes/blocked/+page.svelte

```diff
@@ -0,0 +1,4 @@
+<script lang="ts">
+</script>
+
+<h1>Seems that you have been blocked</h1>
```

Comment:
I think we need a more descriptive text here and give the user a reason. Maybe even add a list of blocked countries?